functional/tdx/run.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+script_path=$(dirname "$0")
+source "${script_path}/../../lib/common.bash"
+
+runtime_type="io.containerd.kata.v2"
+config_file=""
+kernel_tdx_path="/usr/share/kata-containers/vmlinuz-tdx.container"
+qemu_tdx_path="/usr/local/bin/qemu-system-x86_64"
```

Comment:
How about adding a `setup()` function that checks to ensure the architecture is correct "just in case"?